index.js

```diff
@@ -16,8 +16,18 @@ const StartLedger = typeof process.env.LEDGER === 'undefined' ? 32570 : parseInt
 console.log('Fetch XRPL transactions into Google BigQuery')
 
 const Client = new XrplClient(XRPLNodeUrl)
+
+async function safeHalt() {
+  try {
+    await Client.close()
+  } catch(e) {
+    console.error('ERROR closing connection:', e)
+  } finally {
+    process.exit(1)
+  }
+}
   
-Client.ready().then(Connection => {
+Client.ready().then(() => {
```

Comment:
removed this since I know understand that `Client.ready` returns `Client` anyway and we have that variable in global scope here. fewer variables is easier to read.